server/admin_management/api/app/schemas.py

```diff
@@ -215,3 +216,48 @@ class FamAccessControlPrivilegeCreateResponse(BaseModel):
     error_message: Optional[str] = None
 
     model_config = ConfigDict(from_attributes=True)
+
+
+# ------------------------------------- FAM Admin User Access ---------------------------------------- #
+class FamApplicationDto(BaseModel):
```

Comment:
Hi Ian, just wondering if `FamApplicationDto` will feel like used by the FAM application services? same as the `FamRoleDto`, maybe something like `FamGrantApplicationDto`? cause you also use grant in other schemas under this section

Reply:
Actually, that `**FamApplicationDto**` and `**FamRoleDto**` was my intention for being common use, not particularly for "grant" endpoint. I was trying some time 2 days ago to see if I could reuse previous (**FamApplicationBase** and FamRoleBase) with aliases we want ("id", not "application_id" etc...) but found it is difficult to do without not being confusing. So I decided for now create two new Dto(s) for easier implementation.
Later on, I intend to unify them all together when I come back(so make more sense but not necessary database entity naming; but will impact other endpoints and frontend).